accelerator/cuda_accelerator.py

```diff
@@ -26,7 +26,7 @@ def __init__(self):
 
         for _, module_name, _ in pkgutil.iter_modules([os.path.dirname(op_builder_module.__file__)]):
             # avoid self references
-            if module_name != 'all_ops' and module_name != 'builder':
+            if module_name != 'all_ops' and module_name != 'builder' and module_name != 'cpu':
```

Comment:
Currently CPU op_builders is under directory `op_builder/cpu`.  iter_modules for CUDA op_builder will see `cpu/` directory.  Filter it out to avoid enumerate CPU op builders.